index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="#title">Dr Talla Germaine</h1>
+      <p>Hello World!</p>
+      <figure id="img-div">
+        <img id="image" src="https://img.freepik.com/free-vector/doctor-character-background_1270-84.jpg?w=2000" alt=""/>
+        <figcaption id="img-caption">This image is not exactly Dr Talla but is just a representation</figcaption>
+      </figure>
+      <div id="tribute-info">
+        <p>Meet Dr Talla Germain a Gastroenterologist</p>
+        <a id="tribute-link" href="https://www.webmd.com/ibs/what-gastroenterologists-do" target="_blank">Want to know more about a Gastroenterologist?</a>
+      </div>
+      </main>
+   
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+  </body>
+</html>
```

Comment:
Same here, please fix the indentation

Reply:
- [Optional ] please be polite and clear about the inline comment you give feedback on, and explain why it required changing aspects. 👍